apps/phone/src/index.tsx

```diff
@@ -7,6 +7,7 @@ import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
 import updateLocale from 'dayjs/plugin/updateLocale';
 import { Theme as MaterialUITheme } from '@mui/material';
+import { RewriteFrames } from '@sentry/integrations';
```

Comment:
Unused import

Reply:
I never added this.. this was from the PR before mine. Ill remove it though as it shouldn't be there

Reply:
removed